README.md

```diff
@@ -68,53 +70,51 @@ jobs:
 
 The webhook is triggered at least 15 minutes from when there is no translation change. This means whenever translation is added or edited a webhook request is scheduled to be sent to all configured endpoints for the project 15 minutes later. The scheduled request will be sent as planned unless there is a new change to translation before the scheduled time, in which case it is rescheduled to be sent 15 minutes later.
 
-The endpoint should be configured to match the following `Github REST API` post request:
-
-```shell
-curl -v -H "Accept: application/vnd.github.everest-preview+json" \
-        -H "Authorization: token $PERSONAL_ACCESS_TOKEN" \
-        https://api.github.com/repos/:owner/:repo/dispatches \
-        -d '{"event_type":"applanga-pull"}'
-```
-
 Here are the steps to setup the **Webhook Endpoint**
 * Login to Applanga dashboard and navigate to the project. Then click on **Project Settings**
 
 ![](https://www.applanga.com/assets/images/docu/groups_editapp.png)
 
-* In the settings page scroll down to the section **WEB HOOKS** and click the **Add endpoint** button, this will show a modal
-where the endpoint values can be entered
+* In the settings page scroll down to the section **WEB HOOKS** and click the **Add endpoint** button, this will show a modal where the endpoint values can be entered
 ![](https://www.applanga.com/assets/images/docu/webhook_settings.png)
 
-The values should be configured to match the preceding curl request. To be sure, verify the following
-- **POST** is selected as http method
-- **JSON** is selected in the body tab
-- `{"event_type":"applanga-pull"}` is entered in the **Request body**
+* Set the http method to *POST* and enter the endpoint url as follows `https://api.github.com/repos/<OWNER>/<REPO>/actions/workflows/<WORKFLOW_ID>`. The following values `<OWNER>`, `<REPO>` and `<WORKFLOW_ID>` should be replaced with the correct values. The `WORKFLOW_ID` should be the name of the workflow file you want to trigger, in this case `applanga-pull.yml`.
+
+Please refer to the following screenshot 
 
-![](https://www.applanga.com/assets/images/docu/webhook_endpoint_header.png)
+![](https://www.applanga.com/assets/images/docu/webhook_branch_trigger_endpoint_url.png)
 
-![](https://www.applanga.com/assets/images/docu/webhook_endpoint_body.png)
+* **Headers** You need to set the following headers, `Authorization` and `Accept`. The `Authorization` header value is a valid github access token with repository permissions combined like so `token <ACCESS TOKEN>`. For example if you had an access token `ghp_zWAdtUqmtFTY7qkqJS1wmuEx8ytX0SpIPv` then the Authorization header would be set like so `token ghp_zWAdtUqmtFTY7qkqJS1wmuEx8ytX0SpIPv`. Please refer to the following github [documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) on how to generate an access token.
```

Comment:
use <YOUR_API_TOKEN> instead of <ACCESS%20TOKEN> wildcards typically should not have spaces and we use <YOUR_API_TOKEN> in another part of the doc too please double check if namings also for <REPO> and <OWNER> are used consistently throughout the doc